easybuild/easyconfigs/o/OMA/OMA-2.5.0-GCCcore-10.2.0.eb

```
@@ -0,0 +1,38 @@
+# This file is an EasyBuild reciPY as per https://easybuilders.github.io/easybuild/
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics 
+
+easyblock = 'PackedBinary'
+
+name = 'OMA'
+version = '2.5.0'
+
+homepage = 'https://omabrowser.org/standalone/'
+description = """The OMA ('Orthologous MAtrix') project is a method and database for the inference 
+ of orthologs among complete genomes"""
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+
+source_urls = ['http://omabrowser.org/standalone/']
+sources = ['%(name)s.%(version)s.tgz']
+checksums = ['30f5f4e73152c07a61a811ce982bdba2a5e1d236312cc16faa0b388b759e937e']
+
+unpack_options = '--strip-components=1'
+
+install_cmd = 'sh ./install.sh %(installdir)s'
+
+builddependencies = [
+    ('Python', '3.8.6'),
```

@sdx23 with the toolchain change, you will need to add `binutils` as a dependency.

```suggestion
    ('Python', '3.8.6'),
    ('binutils', '2.35'),
```